src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/IdentifiersShouldNotContainUnderscores.Fixer.cs

```diff
@@ -13,17 +20,73 @@ public abstract class IdentifiersShouldNotContainUnderscoresFixer : CodeFixProvi
     {
         public sealed override ImmutableArray<string> FixableDiagnosticIds => ImmutableArray<string>.Empty;
```

Comment:
@Youssef1313 This is your problem! Took me a while to find out what was broken xD

```suggestion
        public sealed override ImmutableArray<string> FixableDiagnosticIds { get; } = ImmutableArray.Create(IdentifiersShouldNotContainUnderscoresAnalyzer.RuleId);
```

Reply:
@mavasani What about having an analyzer that reports on empty list of diagnostic IDs (for analyzers and fixers)?